vllm/config.py

```diff
@@ -150,7 +150,7 @@ def _verify_tokenizer_mode(self) -> None:
     def _verify_sparsity(self) -> None:
         supported_sparsity = ["sparse_w16a16", "semi_structured_sparse_w16a16"]
 
-        if self.quantization is not None:
+        if self.quantization is not None and self.sparsity is not None:
```

Comment:
`_verify_sparsity()` is called for every model load even if the model is not sparse. This is the same thing that happens for quantization. So, this will get triggered for marlin models and trip a value error